python/pyproject.toml

```diff
@@ -1,18 +1,27 @@
-import logging
-from setuptools import find_packages, setup
-from src.aac import __version__
-from os import linesep
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
 
-README_FILE_PATH = "../README.md"
-
-try:
-    with open(README_FILE_PATH, "r") as file_handler:
-        readme_description = file_handler.read()
-except Exception as exception:
-    logging.error(f"Failed to open readme file {README_FILE_PATH} with error:{linesep}  {exception}")
-    readme_description = ""
+[project]
+name = "aac"
+dynamic =  ["version"]
```

Comment:
I'm not seeing where you tell this to look for pulling the version from elsewhere in the codebase?

Reply:
You are right, thank you for catching that, I added the table for dynamic variables.  I suspect theres a simpler way to do it, but this is the only method I've found that works for the setup we have right now.

Reply:
I wonder if this could be part of what was causing the pipeline spaz out? intrigued to see